### PR TITLE
fix build from source instruction

### PR DESCRIPTION
**Pull request by apeforest**
## Description ##
Need to add a blank line before bash section, otherwise the html page is not rendered properly
https://mxnet.apache.org/get_started/ubuntu_setup.html

- break long lines
- remove unnecesarry tab space
- remove $ to make it consistent

@leezu @aaronmarkham please review.

### CODE REVIEW

File: docs/static_site/src/pages/get_started/ubuntu_setup.md [hunk 1]
```diff
@@ -63,11 +63,11 @@ Then download [cuDNN 7.1.4](https://developer.nvidia.com/cudnn).
 Unzip the file and change to the cuDNN root directory. Move the header and libraries to your local CUDA Toolkit folder:
 
 ```bash
-    tar xvzf cudnn-9.2-linux-x64-v7.1
```
Comment (larroy) [inline docs/static_site/src/pages/get_started/ubuntu_setup.md]: Why are we recommending cuda 9.2 in the install instructions?  Most are using 10.1 and 10.2

**Comment by larroy** [comment]
Why do we have 2 build from source pages, this is all extremely confusing.

https://mxnet.apache.org/get_started/ubuntu_setup.html
https://mxnet.apache.org/get_started/build_from_source

File: docs/static_site/src/pages/get_started/ubuntu_setup.md [hunk 2]
```diff
@@ -63,11 +63,11 @@ Then download [cuDNN 7.1.4](https://developer.nvidia.com/cudnn).
...
-    sudo cp -P cuda/include/cudnn.h /usr/local/cuda/include
```
Comment (larroy) [inline docs/static_site/src/pages/get_started/ubuntu_setup.md]: this shouldn't be necessary.

**Comment by apeforest** [comment]
> Why do we have 2 build from source pages, this is all extremely confusing.
> 
> https://mxnet.apache.org/get_started/ubuntu_setup.html
> https://mxnet.apache.org/get_started/build_from_source

This is out of scope of this PR. Please raise this in a seprate PR.

File: docs/static_site/src/pages/get_started/ubuntu_setup.md (hunk 2 above)
Comment (apeforest) [reply docs/static_site/src/pages/get_started/ubuntu_setup.md]: This PR only changes formatting and spacing. For content change, please create a separate PR.

File: docs/static_site/src/pages/get_started/ubuntu_setup.md (hunk 1 above)
Comment (apeforest) [reply docs/static_site/src/pages/get_started/ubuntu_setup.md]: This PR only changes formatting and spacing. For content change, please create a separate PR.

**Comment by leezu** [comment]
@larroy One page contains Ubuntu specific instructions, the other page contains generic instructions.
I think that's reasonable in principle, but the current instructions have a lot of room for simplification and improvement.

**Pull request merged**